### PR TITLE
Path follower migration

### PR DESCRIPTION
**Pull request by lennartalff**
Depends on PR HippoCampusRobotics/path_planning#3

### CODE REVIEW

File: config/CarrotControl.cfg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+PACKAGE = "path_planning"
```
Comment (NBauschmann) [inline config/CarrotControl.cfg:2]: wrong package name

File: launch/path_follower_node.launch [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<launch>
+    <arg name="use_ground_truth" />
+    <arg name="path_file" default="$(find path_planning)/config/infinity_path.yaml"/>
+    <node name="path_follower" pkg="path_planning" type="path_follower_node" output="screen">
```
Comment (NBauschmann) [inline launch/path_follower_node.launch:4]: wrong package name

Comment (lennartalff) [reply launch/path_follower_node.launch:4]: Actually it is the right one.

File: config/CarrotControl.cfg (hunk 1 above)
Comment (lennartalff) [reply config/CarrotControl.cfg:2]: Oh no, how unfortunate

**Review by NBauschmann — APPROVED** [review]

**Pull request merged**